Dockerfile

```diff
@@ -1,41 +1,45 @@
-FROM hashicorp/terraform:0.13.6 AS build-nss-wrapper
-
+ARG REPO
+FROM hashicorp/${REPO} AS build-nss-wrapper
 RUN apk add cmake gcc make musl-nscd-dev libc-dev curl-dev jansson-dev
 RUN git clone git://git.samba.org/nss_wrapper.git
 RUN cd nss_wrapper \
-	&& git checkout nss_wrapper-1.1.11 \
-	&& mkdir obj \
-	&& cd obj \
-	&& cmake -DCMAKE_INSTALL_PREFIX=/usr/local .. \
-	&& make
+        && git checkout nss_wrapper-1.1.11 \
+        && mkdir obj \
+        && cd obj \
+        && cmake -DCMAKE_INSTALL_PREFIX=/usr/local .. \
+        && make
 RUN find / -name libnss_wrapper.so
 
-FROM hashicorp/terraform:0.13.6
+ARG REPO
+FROM hashicorp/${REPO}
 
-ENV \
-	GOPASS_VERSION=1.8.3 \
-	SUMMON_PROVIDER=/usr/local/bin/summon-gopass \
-	SUMMON_VERSION=0.8.3 \
-	TERRAFORM_PROVISIONER_ANSIBLE_VERSION=2.5.0
+# Versions are manually set inside this Dockerfile for the moment!
+ENV GOPASS_VERSION=1.8.3 \
+    SUMMON_PROVIDER=/usr/local/bin/summon-gopass \
+    SUMMON_VERSION=0.8.3 \
+    TERRAFORM_PROVISIONER_ANSIBLE_VERSION=2.5.0
 
 RUN apk add jq gnupg bash ansible
 
 # Install gopass
-RUN wget https://github.com/gopasspw/gopass/releases/download/v${GOPASS_VERSION}/gopass-${GOPASS_VERSION}-linux-amd64.tar.gz -qO - | \
-	tar xz gopass-${GOPASS_VERSION}-linux-amd64/gopass -O > /usr/local/bin/gopass
+RUN wget https://github.com/gopasspw/gopass/releases/download/v${GOPASS_VERSION}/gopass-${GOPASS_VERSION}-linux-amd64.tar.gz
+RUN tar -zxvf gopass-${GOPASS_VERSION}-linux-amd64.tar.gz gopass-${GOPASS_VERSION}-linux-amd64/gopass -O > /usr/local/bin/gopass
```

Comment:
Doing this will create an extra layer in the image so the image will be bigger than needed